packages/mobile/e2e/Transfer.spec.js

```diff
@@ -53,7 +53,12 @@ describe('Transfer Works', () => {
     await element(by.id('JoinCeloContinueButton')).tap()
   })
 
-  it('NUX->Pin', async () => {
+  it.skip('NUX-Terms', async () => {
```

Comment:
Did you mean to leave these .skips here?

Reply:
oh right. 

been having a lot of trouble getting e2e tests running to get these fixed